server/planning/feed_parsers/onclusive_sample.json

```diff
@@ -13,15 +13,9 @@
         "timezoneName": "(EDT) Eastern Daylight Savings Time",
         "timezoneOffset": -4.00
     },
-    "venue": "One King West Hotel & Residence, 1 King St W, Toronto",
+    "venue": "Karuizawa",
     "tbcVenue": false,
-    "venueData": [
-        {
-            "venueId": 20848,
-            "locationLat": 43.648940,
-            "locationLon": -79.378086
```

Comment:
can you also parse those into `location.location.lat` and `location.location.lon`?

Reply:
but as I checked the event response which you shared, there is not any data in `venueData`

Reply:
okay